src/EnergyPlus/HVACManager.cc

```diff
@@ -2761,62 +2621,60 @@ void ReportAirHeatBalance(EnergyPlusData &state)
             // Note - do each Pair a Single time, so must do increment reports for both zones
             //       Can't have a pair that has ZoneA zone number = NumOfZones because organized
             //       in input with lowest zone # first no matter how input in idf
-            if (RefDoorMixing(ZoneLoop).RefDoorMixFlag) { // .TRUE. for both zoneA and zoneB
-                if (RefDoorMixing(ZoneLoop).ZonePtr == ZoneLoop) {
-                    for (int j = 1; j <= RefDoorMixing(ZoneLoop).NumRefDoorConnections; ++j) {
+            if (refDoorMixing.RefDoorMixFlag) { // .TRUE. for both zoneA and zoneB
+                if (refDoorMixing.ZonePtr == ZoneLoop) {
+                    for (int j = 1; j <= refDoorMixing.NumRefDoorConnections; ++j) {
                         //    Capture impact when zoneloop is the 'primary zone'
                         //    that is, the zone of a pair with the lower zone number
-                        if (RefDoorMixing(ZoneLoop).VolRefDoorFlowRate(j) > 0.0) {
-                            int ZoneB = RefDoorMixing(ZoneLoop).MateZonePtr(j);
-                            auto &zoneBHB = state.dataZoneTempPredictorCorrector->zoneHeatBalance(ZoneB);
```

Comment:
[src/EnergyPlus/HVACManager.cc:2771]:(style),[constVariable],Variable 'zoneBHB' can be declared as reference to const